README.md

```diff
@@ -58,6 +58,18 @@ Create file `prometheus/server_targets.yml` and add list of servers to that file
 
 Restart the system everytime you make a change to either of these files.
 
+#### Alert configuration
+
+If you want automatic alerts when a server or service becomes unavailable, do the following:
+
+1. Create file `prometheus/alert_rules.yml`. Start by copying `prometheus/alert_rules.yml.example` and replace `<NMHS>` with relevant name.
+2. Create file `prometheus/alertmanager.yml`. Start by copying `prometheus/alertmanager.yml.example` and replace `<NMHS>` with relevant name. Then: 
+    - configure SMTP fields.
+    - set `email_configs` in `receiver` to the desired address to receive alerts on.
+3. If you wish to receive alerts through other channels, see https://prometheus.io/docs/alerting/latest/configuration/#receiver-integration-settings.
+4. Adjust `prometheus/prometheus.yml` by removing `#` from `alerting` and `rules_files` blocks.
+5. Alter docker compose command to: `docker compose --profile alertmanager up -d`
```

Comment:
Specify where?

Reply:
You mean, where is the original `docker compose`command specified?

Reply:
Given that the previous steps refer to altering some file, this last line gives me the impression that there is a docker compose command listed in a file somewhere, and that this file needs to be updated. Unsure how best to phrase it.

Reply:
Right! I see the problem. I will fix.

Reply:
The update looks far better!